src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -1671,6 +1674,10 @@ namespace FanCoilUnits {
                         }
                     }
                 }
+                ZoneEqSizing(CurZoneEqNum).OAVolFlow = FanCoil(FanCoilNum).OutAirVolFlow;
+                if (FanCoil(FanCoilNum).ATMixerExists) {
+                    SingleDuct::setATMixerSizingProperties(FanCoil(FanCoilNum).ATMixerIndex, ControlledZoneNum, CurZoneEqNum);
+                }
```

Comment:
This function will be used in each zone equipment type to set up ATMixer inlet conditions.